src/lib/plugins/entities.js

```diff
@@ -257,13 +270,20 @@ module.exports.entity=function(entity,serv){
   };
 
   entity.collect = (collectEntity) => {
-    if (entity.type != 'player') serv.emit('error', 'Non-player entity (ttype ' + entity.type + ') cannot collect another entity');
-    else {
+    if (entity.type != 'player'){
+      serv.emit('error', 'Non-player entity (ttype ' + entity.type + ') cannot collect another entity')
+      return;
+    }
+    
+    var EmptySlot = entity.inventory.firstEmptyInventorySlot()
```

Comment:
better call it emptySlot. Identifiers starting by a capital letter are class name.